site/docs/guides/admin/servers/configuring-an-open-lineage-server.md

```diff
@@ -81,7 +81,7 @@ For open lineage server following can be configured:
 
 | Property | Description | Is mandatory |
 |---|---|---|
-`lineageGraphConnection` | OCF configuration object that defines the Graph store connector type used. See [open-lineage-janus-connector](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector) for more details. | Yes |
+`lineageGraphConnection` | OCF configuration object that defines the Graph store connector type used. See [open-lineage-janus-connector](/connectors/runtime/open-lineage-janus-connector) for more details. | Yes |
```

Comment:
isn't the correct link /connectors/governance-daemon/open-lineage-janus-connector (?)